aws_lambda_builders/actions.py

```diff
@@ -99,3 +102,19 @@ def __init__(self, source_dir, dest_dir, excludes=None):
 
     def execute(self):
         copytree(self.source_dir, self.dest_dir, ignore=shutil.ignore_patterns(*self.excludes))
+
+
+class FileCleanupAction(BaseAction):
+
+    NAME = "FileCleanup"
```

Comment:
`RemoveSource`

Not sure RemoveSource is the right name for everything in general.